python/arc/__init__.py

```diff
@@ -1,3 +1,3 @@
 from .sql import enable_arc
 
-__version__ = "0.1.3"
+__version__ = "0.1.4"
```

Comment:
Please match this version in pom file.
Otherwise build in python will fail.
@robertwhiffin